docs/self-managed/deployment/helm/cloud-providers/kind.md

```diff
@@ -0,0 +1,466 @@
+---
+id: kind
+title: "Deploy Camunda 8 to a local kind cluster"
+sidebar_label: "kind"
+description: "Deploy Camunda 8 Self-Managed on a local Kubernetes cluster using kind for development and testing purposes."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+With this guide, you'll deploy Camunda 8 Self-Managed to a local Kubernetes cluster using [kind (Kubernetes in Docker)](https://kind.sigs.k8s.io/). This setup is ideal for learning, development, and testing.
+
+While this guide uses kind, the same concepts apply to other local Kubernetes tools, like [K3s](https://k3s.io/), [minikube](https://minikube.sigs.k8s.io/), or [MicroK8s](https://microk8s.io/). The goal is to reduce the resources required by Camunda 8, so they can run on a personal machine.
+
+:::warning Local development only
+This setup is intended for **local development only** and shouldn't be used in production environments. For production deployments, follow our [cloud provider guides](/self-managed/deployment/helm/cloud-providers/index.md).
+:::
+
+If you encounter issues during deployment, refer to the [Troubleshooting](#troubleshooting) section.
+
+## Deployment modes
+
+You can choose from two deployment modes:
+
+| Mode          | Access                        | Requirements                    | Use case                              |
+| ------------- | ----------------------------- | ------------------------------- | ------------------------------------- |
+| **Domain**    | `https://camunda.example.com` | mkcert, hosts file modification | Full TLS setup, realistic environment |
+| **No-domain** | `localhost` via port-forward  | None                            | Quick setup, minimal configuration    |
```

Comment:
not entirely true about requirements: none

I see a `hosts file modification` otherwise it won't work either.